packages/wonder-blocks-dropdown/src/components/multi-select.tsx

```diff
@@ -547,7 +564,14 @@ const MultiSelect = (props: Props) => {
         | React.ReactElement<React.ComponentProps<typeof SelectOpener>> => {
         const {noneSelected} = labels;
 
-        const menuContent = getMenuTextOrNode(allChildren);
+        const menuTextOrNode = getMenuTextOrNode(allChildren);
+        const [openerStringValue, openerContent] =
+            maybeExtractStringFromNode(menuTextOrNode);
+
+        if (openerStringValue) {
+            // opener value changed, so let's announce it
+            handleAnnouncement(openerStringValue);
```

Comment:
If I'm understanding correctly, this will announce the message on every render, even when the opener value hasn't changed (for example, every time the dropdown is opened/closed). Would it make sense to only announce it when the opener value changes? We could do that by calling the announcer in a `useEffect` with `selectedValues`, or we could call it whenever we trigger the `onChange` event prop for the MultiSelect! What do you think?

Reply:
I debated on whether to gate this with opener value changes, and ultimately kept it this way because of the bugs in Safari and VoiceOver. It will always announce the value to counteract the clipped and stale value that is announced regardless of the visual state of it!

I also played with setting state for these values and using useEffect and got thrown into an infinite loop. This was the only way I could get it to work with the amount of re-rendering going on.

Reply:
Did moving the announcement to a `useEffect` similar to the SingleSelect have the expected behaviour?

Reply:
I believe so! The last I checked it was working great. I'm still working on initializing the Announcer so I'll do another round of screen reader testing then!